README.md

```diff
@@ -18,10 +18,6 @@ Utility  for request tracing, log enrichment and auditing.
 
 # HTTP Request tracing
 
-## Populate zipkin HTTP headers for [tracing](opentracing.io)
-
-This filter initializes an HTTP header(X-B3-TraceID) for tracing, if not already present. The header is also added in the returned response.  Note that if you are already using another library for propogating headers, this will have no effect.
```

Comment:
What if there are library users who now expect the X-B3-TraceID header to be added by the library? They will not have the header any more in responses?

Reply:
Idea is to make LogFilter rely on other libraries/frameworks to propagate context such as Spring Cloud Sleuth or Splunk Observability.  Besides, I am not aware of any consumers of this library besides our security services.

Reply:
@akarollil, the response header is now added by the tracing library. That's the right place for that functionality. It was only added here as a temporary workaround after Spring Sleuth removed its response header functionality.

Reply:
I understand the change and I like the header injection removal, but just thinking about users of the library who may be depending on the injection behavior and will now miss the header.